src/constants/images.ts

```diff
@@ -0,0 +1,2 @@
+export const juiceboxEmojiImageUri =
+  'https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/160/apple/232/beverage-box_1f9c3.png'
```

Comment:
**suggstion** maybe we can pin this on ipfs

Reply:
Done